_pkgdown.yml

```diff
@@ -0,0 +1,21 @@
+url: https://dfe-analytical-services.github.io/dfeshiny/
+template:
+  bootstrap: 5
+  bootswatch: cyborg
+  bslib:
```

Comment:
Another potential spot for custom CSS is the highlighting used in the citation example, the text is quite light 
![image](https://github.com/dfe-analytical-services/dfeshiny/assets/52536248/a51eeeea-549b-4229-b2a3-68a5e08eb4c0)

Reply:
Fixed now